src/Options/index.js

```diff
@@ -282,6 +282,9 @@ export interface ParseServerOptions {
   /* Set to true if new users should be created without public read and write access.
   :DEFAULT: false */
   enforcePrivateUsers: ?boolean;
+  /* Allow login with expired authData token.
```

Comment:
```suggestion
  /* Allow a user to log in even if the 3rd party authentication token that was used to sign in to their account has expired. If this is set to `false`, then the token will be validated every time the user signs in to their account. This refers to the token that is stored in the `_User.authData` field. Defaults to `true`.
```

Requires `npm run definitions`

Reply:
@Moumouls could you please review 😊

Reply:
i think your suggestion is okay @mtrezza , more detailed. In the futur we need to switch `default: false`

But we can't currently since it could be a huge breaking change.